bindings/nodejs/tests/client/utilityMethods.spec.ts

```diff
@@ -116,14 +116,8 @@ describe('Client utility methods', () => {
         const message = '0x494f5441';
         const signature = await new SecretManager(secretManager).signEd25519(
             message,
-            // [44, 4218, 0, 0, 0] IOTA coin type, first account, first public address
-            [
-                { hardened: true, bs: [128, 0, 0, 44] },
-                { hardened: true, bs: [128, 0, 16, 123] },
-                { hardened: true, bs: [128, 0, 0, 0] },
-                { hardened: true, bs: [128, 0, 0, 0] },
-                { hardened: true, bs: [128, 0, 0, 0] },
-            ],
+            // HD-Wallet type, IOTA coin type, first account, first public address
```

Comment:
Can you maybe slightly change the last item of the comment? Even if correct, the way it's written it feels like an item is missing